priv/templates/boot_check.eex

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
+curenv=($(printenv | sed -e 's/=/="/' | sed -e 's/$/"/'))
```

Comment:
actually I think if we leave this as a plainstring and don't array-ize it, we can just rely on \n separator safely